doc/introduction/measurements.rst

```diff
@@ -99,7 +99,7 @@ the measurement results always coincide, and the lists are therefore equal:
 
 >>> np.all(result[0] == result[1])
 True
-
+ 
```

Comment:
```suggestion

```

Reply:
Not sure why the trailing white space didn't get caught by the black check, but no big deal.